### ISSUE TITLE
out of band meeting to handle objections from upstream

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/modules#297]
Do people have time this week to meet and try and reach consensus about the changes requested from upstream? I would like see if we could resolve the conflicts within our team in a timely fashion and not require another week and a half before we can iterate

Here's a doodle: https://doodle.com/poll/fhv643f6g38y8wn9

**Comment by SMotaal** [comment]
Possible discussion points I would personally consider relevant:

- Feedback on versus preemptive bikeshedding of `--type=…` in `--experimental-modules` (#296)

- Why `-m` would be a good/bad choice (#295 and [ecmascript-modules#64](https://github.com/nodejs/ecmascript-modules/issues/64))

**Comment by GeoffreyBooth** [comment]
We also need to discuss [my/Myles’ proposal](https://github.com/nodejs/modules/issues/296#issuecomment-474102657) regarding `--package-type`, and either get consensus on it or on something else that otherwise addresses the upstream need. It appears that we can’t let a week or two go by without responding to their concerns.

**Comment by MylesBorins** [comment]
@nodejs/modules 

It looks like the best times we have right now and 10 / 11 am ET tomorrow... although @ljharb is unable to make either meeting... since we can't find a single time that has all folks with objections nor quorum I'm not convinced the meeting is going to be very effective.

Based on that I'd like to suggest the following with regard to upstream implementation... mostly implying that we move forward with what upstream has requested, and assume there are no upstream objections we get the branch ready to land Wednesday and then have one last review as a team before we land it (and celebrate).

Based on current upstream objections these would be the changes, compared to what we originally proposed

* removal of the `-m` flag
* renaming `--type` to `--entry-type`

I think it is reasonable for us to move forward with these if we have a basic consensus between all groups right now and then buckle down on quickly iterating as a group on specific changes to the above if we want to make them... we can still get them in before 12.x if we work fast enough.

Thoughts?

**Comment by MylesBorins** [comment]
To be VERY clear. Please explicitly object if you have any issues with the above plan. Our ability to move quickly here is dependent on communication

**Comment by SMotaal** [comment]
@MylesBorins Can I be more explicit, I have no objections on what the team agrees to upstream but would like to go into the meeting to discuss the two points you state above.

Also, I feel we need to allocate a firm timebox to align expectations where needed — ie How objections within and in spite of our process by team members affect our collective efforts and commitments.

**Comment by MylesBorins** [comment]
@SMotaal I'm sorry but that wasn't exactly clear to me. No one seems to be objecting to what we have upstream right now, are you stating you still want to have the out of band meeting?

**Comment by SMotaal** [comment]
@MylesBorins failing to find my good hat for making the right assumptions or at least not misunderstanding your initial post, sure that was my own intent from the post, but always yielding to the teams overall heading — so we are good there no need for apologies and I appreciate how hard it is juggling everything this past week.

I would revise my statement, maybe a separate issue would be a good idea (please advise), I would revise it:

> go into ~~the~~ **next** meeting to discuss the two points you state above, but more importantly, to discuss process expectations and team member responsibilities towards that and their peers.

**Comment by SMotaal** [comment]
@MylesBorins Are we not meeting this week?

**Comment by MylesBorins** [comment]
we are. Making agenda issue right now. Sorry I was so burried I totally spaced that I need to do that 😇

**Comment by SMotaal** [comment]
No worries — I just worried me be distracted and such 👍
